mixnet/node.py

```diff
@@ -48,28 +45,28 @@ def encryption_public_key(self) -> X25519PublicKey:
     def sphinx_node(self) -> Node:
         return Node(self.encryption_private_key, self.addr)
 
-    def start(
+    async def start(
         self,
-        delay_rate_per_min: int,
+        delay_rate_per_min: int,  # Poisson rate parameter: mu
         inbound_socket: PacketQueue,
         outbound_socket: PacketPayloadQueue,
-    ) -> MixNodeRunner:
-        thread = MixNodeRunner(
+    ) -> Tuple[MixNodeRunner, asyncio.Task]:
+        runner = MixNodeRunner(
             self.encryption_private_key,
             delay_rate_per_min,
             inbound_socket,
             outbound_socket,
         )
-        thread.daemon = True
-        thread.start()
-        return thread
+        task = asyncio.create_task(runner.run())
+        return runner, task
```

Comment:
Uhm, if we create a task this do not really need to be `async` as there is no `await`.

Reply:
Absolutely. I fixed it: bd02fb4